frontend/src/storage.js

```diff
@@ -86,7 +86,10 @@ export default class Storage {
   }
 
   getFileById(id) {
-    return this.engine.getItem(id);
+    try {
+    return JSON.parse(this.engine.getItem(id));
```

Comment:
Indenting here looks v wrong. If I had to guess, it's because we've disabled indenting checks in ESLint since that's handled automatically by **prettier**. 

One solution may be to have some auto-format+lint code on a Git hook. It'd force **prettier** formatting on each commit, which may cause 1-2 noisy commits w/ unrelated formatting fixes at first, but then probably 👌 after that initial hiccup.